packages/material-ui-styles/src/ThemeProvider/nested.js

```diff
@@ -1,3 +1,3 @@
-const nested = Symbol('nested');
+const hasSymbol = typeof Symbol === 'function';
 
-export default nested;
+export default (hasSymbol ? Symbol('nested') : '__THEME_NESTED__');
```

Comment:
This is now a little bit different without Symbol. If we support the fallback we need to use `Symbol.for`. Otherwise the value is different across realms (possible between hmr?). React doesn't support multiple instances either and they use Symbol.for. We should probably do the same.

Reply:
This is the approach used by React. I don't know enough to comment, this sounds like a safer approach.

Reply:
So this means I should replace Symbol("nested") to Symbol.for("nested") right?